src/content/resources/blog-images-rss.json

```
@@ -0,0 +1,5 @@
+{
+  "category": "images",
+  "title": "Add blog post images to your Astro RSS Feed",
+  "link": "https://webreaper.dev/posts/astro-rss-feed-blog-post-images/"
```

```suggestion
  "link": "https://webreaper.dev/posts/astro-rss-feed-blog-post-images/"
  "author": "Boston343"
```
Wondering about future-proofing this, in the case you want these to look even fancier e.g. with a Link to the author's GitHub profile or displaying their profile picture.

Fun fact! Just like someone's GitHub profile url is `https://github.com/<username>`, their profile picture is available at `https://github.com/<username>.png`!

So, storing just the username gives you opportunities for whatever aspect of their profile you might want to do something with.

If you'd like this, then I can be tracking these down for you.

---

Yeah this sounds great!
We do want author information, we have a working branch for adding that here https://github.com/astrolicious/astro-tips.dev/pull/79
So I think we should use the same solution, but if we can already track them down, that would be amazing